formSchemas.js

```diff
@@ -0,0 +1,13 @@
+const loginSchema = {
```

Comment:
Will we create a new object for each "page" or for each "section"? (Section being login, personal info, tax info, tax credits vs pages being "enter your access code, enter your SIN, DoB, etc)

Reply:
I think we should 'section' them at this point.

So we would have 

```
const loginSchema = {
  code: {...},
  sin: {...},
  dob: {...}
}
````

Reply:
I imagine for each form, which happens to get broken down into page, but this is up for discussion! As I don't have more forms to work with right now--undecided! but good question

Reply:
yeah, not a big deal for this PR